tests/integration/test_http_post.py

```diff
@@ -0,0 +1,31 @@
+# -*- coding: utf-8 -*-
+# SPDX-License-Identifier: MIT
+
+import utils
+
+
+def test_http_post(test_env):
```

Comment:
Can we name this after the use case?

Reply:
I renamed it to `test_creating_new_request`, is it ok?